docs/en/latest/deployment-modes.md

```diff
@@ -111,23 +111,77 @@ This method is more suitable for two types of users:
+
+2. The API-driven is an emerging paradigm for standalone.
+
+    The routing rules will be entirely in memory and not in a file, and it will need to be updated using the dedicated Standalone Admin API.
+
+    I.e. we need to send an HTTP PUT request to this API containing the configuration in JSON or YAML format, which will flush the configuration used by each worker in the current APISIX instance.
+
+    Changes will overwrite the entire configuration and take effect immediately without requiring a reboot, as it is hot updated.
+
+    This requires us to set the APISIX role to traditional (since we need to start both the API gateway and the Admin API endpoint) and use the yaml config provider. That is, set `deployment.role` to `traditional` and `deployment.role_traditional.config_provider` to `yaml`.
+
+    Refer to the example below:
+
+    ```yaml
+    deployment:
+      role: traditional
+      role_traditional:
+        config_provider: yaml
+    #END
+    ```
+
+    This disables the local file source of configuration in favor of the API. When APISIX starts, it uses the empty configuration until you update it via the API.
+
+    The following are API endpoints:
+
+    ```text
+    ## Update configuration
+    ## The conf_version is not required, if it is not entered by the client, the current 10-digit epoch time is used by default.
+    curl -X PUT http://127.0.0.1:9180/apisix/admin/configs?conf_version=1234 \
+        -H "X-API-KEY: <apikey>"
+        -H "Content-Type: application/json" ## or application/yaml
+        --data-binary @config.json
+
+    ## Get latest configuration
+    curl -X GET http://127.0.0.1:9180/apisix/admin/configs
+        -H "X-API-KEY: <apikey>"
+        -H "Accept: application/json" ## or application/yaml
+    ```
+
+    The update API validates the input and returns an error if it is invalid. If the configuration is accepted, it responds with a `202 Accepted` status and includes the latest configuration version in the `X-APISIX-Conf-Version` header.
+
+    The get API also returns the version number via the `X-APISIX-Conf-Version` header, and returns a response body containing the configuration in a specific format as requested by the client `Accept` header.
+
+    These APIs apply the same security requirements as the Admin API  — such as API key, TLS/mTLS, CORS, and IP allowlist — no changes or additions.
+
+    The API accepts input in the same format as the file-based mode described above, although it also allows the user to input JSON instead of just YAML. The following example still applies. However, the API does not rely on the `#END` suffix because HTTP will guarantee input integrity.
+
+    *Note*: In this case, the Admin API based on etcd is not available, the configuration can only be flushed as a whole, rather than modifying a part of it, and the client needs to build request containing the complete new configuration to be sent to the API.
```

Comment:
```suggestion
    *Note*: In this case, the Admin API based on etcd is not available. The configuration can only be flushed as a whole, rather than modified partially, and the client must send a request containing the complete new configuration to the API.
```